{
  "stats": {
    "suites": 0,
    "tests": 1,
    "passes": 0,
    "pending": 0,
    "failures": 1,
    "start": "2021-04-09T11:13:42.709Z",
    "end": "2021-04-09T11:13:43.068Z",
    "duration": 359,
    "testsRegistered": 1,
    "passPercent": 0,
    "pendingPercent": 0,
    "other": 0,
    "hasOther": false,
    "skipped": 0,
    "hasSkipped": false
  },
  "results": [
    {
      "uuid": "b2da5a79-1b8e-4b14-a0c8-ca8524909b10",
      "title": "",
      "fullFile": "cypress/integration/UI/error.js",
      "file": "cypress/integration/UI/error.js",
      "beforeHooks": [],
      "afterHooks": [],
      "tests": [
        {
          "title": "An uncaught error was detected outside of a test",
          "fullTitle": "An uncaught error was detected outside of a test",
          "timedOut": null,
          "duration": 320,
          "state": "failed",
          "speed": null,
          "pass": false,
          "fail": true,
          "pending": false,
          "context": null,
          "code": "throw err;",
          "err": {
            "message": "ReferenceError: The following error originated from your test code, not from Cypress.\n\n  > Given is not defined\n\nWhen Cypress detects uncaught errors originating from your test code it will automatically fail the current test.\n\nCypress could not associate this error to any specific test.\n\nWe dynamically generated a new test to display this failure.",
            "estack": "ReferenceError: The following error originated from your test code, not from Cypress.\n\n  > Given is not defined\n\nWhen Cypress detects uncaught errors originating from your test code it will automatically fail the current test.\n\nCypress could not associate this error to any specific test.\n\nWe dynamically generated a new test to display this failure.\n    at Object.1 (http://localhost:54150/__cypress/tests?p=cypress/integration/UI/error.js:5:1)\n    at o (http://localhost:54150/__cypress/tests?p=cypress/integration/UI/error.js:1:265)\n    at r (http://localhost:54150/__cypress/tests?p=cypress/integration/UI/error.js:1:431)\n    at eval (http://localhost:54150/__cypress/tests?p=cypress/integration/UI/error.js:1:460)\n    at eval (<anonymous>)",
            "diff": null
          },
          "uuid": "a52c43bb-b534-4276-a366-90178f3ac782",
          "parentUUID": "b2da5a79-1b8e-4b14-a0c8-ca8524909b10",
          "isHook": false,
          "skipped": false
        }
      ],
      "suites": [],
      "passes": [],
      "failures": [
        "a52c43bb-b534-4276-a366-90178f3ac782"
      ],
      "pending": [],
      "skipped": [],
      "duration": 320,
      "root": true,
      "rootEmpty": false,
      "_timeout": 2000
    }
  ],
  "meta": {
    "mocha": {
      "version": "7.0.1"
    },
    "mochawesome": {
      "options": {
        "quiet": false,
        "reportFilename": "mochawesome",
        "saveHtml": false,
        "saveJson": true,
        "consoleReporter": "spec",
        "useInlineDiffs": false,
        "code": true
      },
      "version": "6.2.2"
    },
    "marge": {
      "options": {
        "id": "default",
        "reportDir": "cypress/reports/mocha",
        "quite": true,
        "overwrite": false,
        "html": false,
        "json": true
      },
      "version": "5.2.0"
    }
  }
}